### DISCUSSION TITLE
German Translation

### DISCUSSION THREAD

**Discussion by FineFindus** [GradienceTeam/Gradience#88] [General]
After providing a first translation for German, I received the following feedback (Thanks again). As I still have a few questions about the 
feedback, I wanted to discuss it, and more feedback here.

- [x] ~The first long translation does not make sense at all.~ I think it should be fixed now.
- [ ] The second long one is missing a space after the closing tag (I'm not sure what exactly is meant here; The English translation does not have a space after the closing tag)
- [x] ~"Save preset as..." once uses ... And once the three dots in one character.~ Changed it to use three single dots instead of U+2026
- [x] ~The "Saving preset to..." is a little wrong at the end.~ Should be fixed now.
- [x] I would guess translator-credits would need the names of the translators? (That makes much more sense, I wasn't quite sure what it was. Is there any special format I have to pay attention to? And who should be noted there? All translators or just the ones for a specific language?

I hope somebody can help me with these questions, so I can fix the rest of the translations.

**Comment by ArtyIF** [comment]
- The second long one - I'm not sure what you're talking about. But I noticed that "Benannte Farben werden von einige Apps genutzt. Standardfarben sind in den {1}GNOME Human Interface Guidelines{1} definiert." doesn't have a space before "definiert".
- I added a dev comment to the translator-credits field. You should only note the translators that worked for the current language. If you want to put a URL, just add it after your name, if you want to put an email, add it after your name in \<angle brackets> (just not simultaneously with URL). If there's multiple translators, separate them with new lines.

**Reply by FineFindus** [reply]
I believe the second long one could be this one (It's one with a tag):
> Saving preset to {0}. If that preset already exists, it will be overwritten!
But it doesn't have a space after the tag.

I fixed the one with GNOME HIG, it now has a space before `definiert`, thanks for catching it.
As for the translator credits, as far as I'm aware, I'm currently the only German translator, so I just added my name. Thanks for the help with the format. I think a dev comment was definitely needed to explain it a bit.
